README.md

```diff
@@ -517,6 +518,60 @@ The SAP BTP service operator project maintainers will respond to the best of the
 
 [Back to top](#sap-business-technology-platform-sap-btp-service-operator-for-kubernetes)
 
+## Secret Formats
+
+### Default
+```bash
+#Credentials section
```

Comment:
Credentials Section

Reply:
I want to be clear that this is a comment